sqlx-core/src/postgres/options/pgpass.rs

```diff
@@ -89,11 +89,17 @@ fn load_password_from_line(
     database: Option<&str>,
 ) -> Option<String> {
     let whole_line = line;
-    matches_next_field(whole_line, &mut line, host)?;
-    matches_next_field(whole_line, &mut line, &port.to_string())?;
-    matches_next_field(whole_line, &mut line, username)?;
-    matches_next_field(whole_line, &mut line, database.unwrap_or_default())?;
-    Some(line.to_owned())
+
+    match line.trim().chars().next() {
```

Comment:
`.trim_start()` is slightly more efficient since we don't care about whitespace at the end of the line for this check.